pkg/openstack/glance.go

```diff
@@ -23,17 +24,32 @@ const (
 	// label set by the service operator as in case of split glanceAPI type the
 	// the label on public svc gets set to -external and internal instance svc
 	// to -internal instead of the glance top level glanceType split
-	svcSelector = "tlGlanceAPI"
+	svcSelector   = "tlGlanceAPI"
+	targetVersion = "v18.0.9"
```

Comment:
I'm not sure about `targetVersion`: it might work for a downstream release but not for an upstream one!

Reply:
should we add const https://github.com/openstack-k8s-operators/openstack-operator/blob/main/apis/core/v1beta1/openstackversion_types.go which reflects the mapping of FR releases to version (not sure if MR release would also be required)? with this we could just say here targetVersion < FR3? for upstream we also have those OPENSTACK_RELEASE_VERSION 0.2.0 (fr2 - https://github.com/openstack-k8s-operators/openstack-operator/blob/18.0-fr2/Makefile#L6) 0.3.0 (fr3 - https://github.com/openstack-k8s-operators/openstack-operator/blob/main/Makefile#L6). we'd only have to verify if this is properly handled in the update ci job.